README.md

```diff
@@ -1,2 +1,15 @@
 # Android_Kotlin_D2_Practice1_cats_dogs
 RecyclerView multi-type item, https://dog.ceo/dog-api/ , https://thecatapi.com/,
+
+<img src="https://github.com/vasilevkin/Android_Kotlin_D2_Practice1_cats_dogs/blob/practice1/Screenshot%202020-02-26%20at%2022.12.25.png" width="200">
```

Comment:
.gitignore (example https://github.com/github/gitignore/blob/master/Android.gitignore)

Reply:
This text in the readme, it should be under version control.